pkg/webreg/webreg.go

```diff
@@ -729,6 +729,78 @@ to configure that test to run on a schedule, instead of as a pre-submit:
 Note that the build farms used to execute jobs run on UTC time, so time-of-day based
 <code>cron</code> schedules must be set with that in mind.
 </p>
+
+<h3 id="image-references"><a href="#image-references">Referencing Images</a></h3>
+<p>
+As <code>ci-operator</code> is OpenShift-native, all images used in a test workflow
+are stored as <code>ImageStreamTags</code>. Inside of the <code>Namespace</code> which
+contains a test workflow, the following <code>ImageStreams</code> will exist:
+</p>
+
+<table>
+  <tr>
+    <th style="white-space: nowrap"><code>ImageStream</code></th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td style="white-space: nowrap"><code>pipeline<code></td>
+    <td>Input images described with <code>base_images</code> and <code>build_root</code> as well as images holding built artifacts and output images as defined in <code>images</code>.</td>
```

Comment:
...images holding built artifacts... is quite vague, maybe clarify it with an example, e.g. 

...images holding built artifacts (such as `src` or `bin`)